File: ML/Ayushsingh07_Ayushkumarsingh_2024cse1037_2/gettingstarted/printallprimetillN.py
import math
low=int(input())  
high=int(input())  
  
for num in range(low,high+1):  
   if num==1:
       print(num)
   if num > 1:
        for i in range(2,int(math.sqrt(num)+1)):  
            if (num % i)==0:  
                break  
        else:  
            print(num) 

File: Web/kartikmehta8_KartikMehta_2024csit1039_2/Getting Started/printAllPrimeTillN.cpp

#include <iostream>
#include <cmath>
using namespace std;

int isPrimeNumber(int X) {
    if (X <= 1) {
        return 0;
    }
    for (int i = 2; i <= sqrt(X); i++) {
        if (X % i == 0) {
            return 0;
        }
    }
    return 1;
}

int main()
{
    int low, high, isPrimeNumber(int X);
    cin >> low >> high;
    for (int i = low; i <= high; i++) {
        if (isPrimeNumber(i) == 1) {
            cout << i << "\n";
        }
    }
    return 0;
}
